File: C++/Kids With the Greatest Number of Candies.cpp
class Solution {
public:
    vector<bool> kidsWithCandies(vector<int>& candies, int extraCandies) {
        int max=*max_element(candies.begin(),candies.end());
        vector<bool>v;
        for(auto i: candies){
            if(i+extraCandies>=max){
                v.push_back(true);
            }else{
                 v.push_back(false);
            }
        }
            candies.clear();
            return v;
    }
};

File: JS/Kids With the Greatest Number of Candies.js
/**
 * @param {number[]} candies
 * @param {number} extraCandies
 * @return {boolean[]}
 */
var kidsWithCandies = function(candies, extraCandies) {
    let ans=[];
    let max=Math.max(...candies)
    console.log(max);
    for(let i=0;i < candies.length;++i){
        if(candies[i]+extraCandies >= max){
            ans.push(true);
        }else{
            ans.push(false);
        }
    }
    return ans;
};